File: app/src/main/java/org/lemonadestand/btb/features/login/models/Description.java
package org.lemonadestand.btb.features.login.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class Description {
    @SerializedName("type")
    String type;

    @SerializedName("content")
    List<Content> content;


    public void setType(String type) {
        this.type = type;
    }
    public String getType() {
        return type;
    }

    public void setContent(List<Content> content) {
        this.content = content;
    }
    public List<Content> getContent() {
        return content;
    }


}

File: app/src/main/java/org/lemonadestand/btb/core/models/Description.kt
package org.lemonadestand.btb.core.models

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.parcelize.Parcelize

@Parcelize
data class Description(
	@SerializedName("type")
	var type: String? = null,

	@SerializedName("content")
	var content: List<Content>? = null,
) : Parcelable
